internal/pkg/service/buffer/worker/service/file_test.go

```diff
@@ -65,6 +65,7 @@ func TestUploadAndImportE2E(t *testing.T) {
 
 	// Create 6 records - trigger the slice upload (>=5)
 	for i := 1; i <= 6; i++ {
+		time.Sleep(time.Millisecond)
 		assert.NoError(t, api.Import(apiDeps, &buffer.ImportPayload{
```

Comment:
Fixnuty nestabilny test.

2 records sa vygenerovali v tu istu milisekundu (v teste sa pouzivaju real clock).

V `key` je cas s presnostou na milisekundy.

Vo vystupe mozu byt teda zoradene nahodne, podla `randomSuffix`.

V produkcii ti to nevadi, lebo prisli "naraz".

Ale v teste su `payloads` cislovane, takze musime zabezpecit poradie.

![image](https://user-images.githubusercontent.com/19371734/215991698-8a140ed1-678d-4583-b01e-fe1aa12a6cec.png)